views/App.jsx

```diff
@@ -20,31 +21,46 @@ const App = (props) => {
   const [openSidebar, setOpenSidebar] = useState(true)
   const [notification, setNotification] = useState(NOTIFICATION_DEFAULT)
   const [user, setUser] = useState(null)
+  const [width, setWidth] = useState(0)
+
   const handleNotificationClose = () => setNotification(NOTIFICATION_DEFAULT)
+  const handleResize = () => setWidth(window.innerWidth)
+
+  useEffect(() => {
+    handleResize()
+
+    window.addEventListener('resize', handleResize)
+
+    return () => window.removeEventListener('resize', handleResize)
+  }, [])
 
   return (
     <MuiThemeProvider>
       <CssBaseline />
       <ThemeContext.Provider
         value={{ classes: props.classes, theme: props.theme }}
       >
-        <SidebarContext.Provider value={[openSidebar, setOpenSidebar]}>
-          <NotificationContext.Provider value={[notification, setNotification]}>
-            <ConfigurationsContext.Provider
-              value={[configurations, setConfigurations]}
+        <DimensionsContext.Provider value={[width, setWidth]}>
```

Comment:
I'm not really clear why we need this, is there some issue we can't resolve with CSS?